Pod/Classes/CDTQQueryValidator.m

```diff
@@ -287,28 +349,46 @@ + (BOOL)validateClause:(NSDictionary *)clause
     //$exits lt
 
     NSArray *validOperators =
-        @[ @"$eq", @"$lt", @"$gt", @"$exists", @"$not", @"$ne", @"$gte", @"$lte" ];
+        @[ @"$eq", @"$lt", @"$gt", @"$exists", @"$not", @"$gte", @"$lte", @"$in" ];
 
     if ([clause count] == 1) {
         NSString *operator= [clause allKeys][0];
 
-    if ([validOperators containsObject:operator]) {
-        // contains correct operator
-        id clauseOperand = [clause objectForKey:[clause allKeys][0]];
-        // handle special case, $notis the only op that expects a dict
-        if ([operator isEqualToString:@"$not"] && [clauseOperand isKindOfClass:[NSDictionary class]]) {
-            return [CDTQQueryValidator validateClause:clauseOperand];
-
-        } else if ([CDTQQueryValidator validatePredicateValue:clauseOperand
-                                                  forOperator:[clause allKeys][0]]) {
-            return YES;
+        if ([validOperators containsObject:operator]) {
+            // contains correct operator
+            id clauseOperand = [clause objectForKey:[clause allKeys][0]];
+            // handle special case, $notis the only op that expects a dict
+            if ([operator isEqualToString:NOT]) {
+                return [clauseOperand isKindOfClass:[NSDictionary class]] &&
+                       [CDTQQueryValidator validateClause:clauseOperand];
+
+            } else if ([operator isEqualToString:IN]) {
+                return [clauseOperand isKindOfClass:[NSArray class]] &&
+                       [CDTQQueryValidator validateInListValues:clauseOperand];
```

Comment:
I removed the `In` in method names in both the Validator and the Translator.  To me the `In` still makes sense because it is the `IN` list we are dealing with but I suppose I see your point.  Changed in b667c92d0cf9994ca7a8751af6e7efedcb6f3199